### ISSUE TITLE
Link to the quote as a part of QuoteDTO

### ISSUE THREAD

**Issue by rexim** [codingteam/loglist#99]

**Comment by ForNeVeR** [comment]
I don't think we'll ever need this. API client won't be interested in a public link, because a graphical API client will have its own to present the quote. Id should be enough for any kind of client.

@rexim, do you remember the reasoning behind this issue?

**Comment by rexim** [comment]
@ForNeVeR yes, I wanted to use it for Horta Hell, so she can print the public link without worring if anything changed in the public schema.

**Comment by ForNeVeR** [comment]
Alright, I have no additional questions for now.
